apps/right-to-rent-check/behaviours/age-restriction.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const moment = require('moment');
+
```

Comment:
I couldn't just use [`before` argument in the config](https://github.com/UKHomeOfficeForms/hof-examples/blob/master/apps/date-component/README.md#date-argument) because I'm already using it for future DOB.  It's a pity you can't just reuse before

Also, I explored the possibility of just using under 18 validation for future dates but this was voted ⬇️ by research and design peeps